data/formative-questionnaires/module-4.yml

```diff
@@ -0,0 +1,700 @@
+        correct_answers:
+          - 1
+
+  4-2-1-5b:
+    questions:
+      1_communication_frustration:
+        multi_select: false
+        label: |
+          Which of the following can support children to manage communication frustration?
+        assessment_summary: |
+          If you make sure that the daily routine and your behaviour are predictable it will support children to feel safe and secure.
+          This will help them to feel confident.
+          If children feel confident and settled, their reactions to frustrations will be minimised.
+        assessment_fail_summary: |
+          If you make sure that the daily routine and your behaviour are predictable it will support children to feel safe and secure.
+          This will help them to feel confident.
+          If children feel confident and settled, their reactions to frustrations will be minimised.
+        answers:
+          1: Highlighting negative behaviour
+          2: Having predictable behaviour and routines
+          3: Limited knowledge of children’s needs
+          4: Limited use of PECS
+        correct_answers:
+          - 2
+
+  4-2-3-1b:
+    questions:
+      1_pandemic_positive:
+        multi_select: false
+        label: |
+          Which of the following can be a positive outcome of the pandemic lockdowns?
+        assessment_summary:  |
+          Children with older siblings may have had access to resources designed for older children.
+          They may also have experienced and rehearsed a more advanced vocabulary.
+        assessment_fail_summary: |
+          Children with older siblings may have had access to resources designed for older children.
+          They may also have experienced and rehearsed a more advanced vocabulary.
+        answers:
+          1: Limited outdoor space
+          2: Having access to resources belonging to older siblings
+          3: Limited interaction
+          4: Increased screen time
+        correct_answers:
+          - 2
+
+  4-2-3-2b:
+    questions:
+      1_pandemic_negative:
+        multi_select: false
+        label: |
+          Which of these may have had a negative impact on children’s language skills during the pandemic lockdowns?
+        assessment_summary: |
+          Children may have had increased access to technology including computers, TVs and phones.
+          Whilst this means that children will have heard vocabulary, it is not as effective as the interactions that take place between people.
+        assessment_fail_summary: |
+          Children may have had increased access to technology including computers, TVs and phones.
+          Whilst this means that children will have heard vocabulary, it is not as effective as the interactions that take place between people.
+        answers:
+          1: Musical activity
+          2: Outdoor activity
+          3: Interactions
+          4: Screen time
+        correct_answers:
+          - 4
+
+# ---------------
```

Comment:
@helenrappand @JPendriss @laura-biscuits I have updated the questions above, but not beyond.